bitnami/aspnet-core/Chart.yaml

```
@@ -31,4 +31,4 @@ maintainers:
 name: aspnet-core
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/aspnet-core
-version: 6.0.3
+version: 6.0.4
```

We need to bump the version again
```suggestion
version: 6.0.5
```